docs/README.md

```diff
@@ -159,3 +159,8 @@ For more information see [custom commands documentation](custom.md)
 - [flow: feature/release/hotfix publish](flow.md#flow-featurereleasehotfix-publish)
 - [flow: feature/release track](flow.md#flow-featurerelease-track)
 - [flow: feature pull](flow.md#flow-feature-pull)
+
+
+## Package Testing
+
+Check the implementation details of the [tests](testing.md).
```

Comment:
This should be moved into `/CONTRIBUTE.md`, since this is not pertinent for GitSavvy users.